server/src/main/java/org/elasticsearch/cluster/coordination/NodeJoinExecutor.java

```diff
@@ -300,6 +307,13 @@ public boolean runOnlyOnMaster() {
         return false;
     }
 
+    void blockForbiddenVersions(TransportVersion joiningTransportVersion) {
+        if (FORBIDDEN_VERSIONS.contains(joiningTransportVersion)) {
+            throw new IllegalStateException("A node with transport version " + joiningTransportVersion + " is forbidden from joining this cluster"
```

Comment:
Perhaps we could tweak the wording so the line doesn't split just the closing paren :)

Reply:
forgot spotless :(